backend/models.py

```diff
@@ -24,15 +46,18 @@ class SharedUpdate(SQLModel):
 class BookBase(SharedBase):
     title: str = Field(index=True)
     author: str = Field(index=True)
-    synopsis: Optional[str] = None
-    edition: Optional[str] = None
+    abstract: Optional[str] = None
+    publisher: Optional[str] = None
     catalog: Optional[str] = Field(default=None, index=True)
     category_type: Optional[str] = Field(default=None, index=True)
     category_age: Optional[str] = Field(default=None, index=True)
     category_topics: Optional[str] = None
-    langage: Optional[str] = None
-    cover: Optional[str] = None
+    langage: Optional[LanguageAlpha2] = None
```

Comment:
which writing should we take: langage or language?

Reply:
language is correct